vitkon/form-container
# Removes yarn in favour of package-lock ☠️

#### What's this PR do?
Removes `yarn.lock` in favour of `package-lock`

#### Where should the reviewer start?
`package.json`
  
#### How should this be manually tested?
remove node_modules, npm install and make sure that you get a deterministic deps in `node_modules` folder
  
#### What are the relevant tickets / issues?
#20
